README.md

```diff
@@ -190,9 +190,11 @@ Clone this repository, `cd` into the folder created, then install dependencies:
 ```sh
 git clone https://github.com/bitcrowd/bitstyles.git
 cd bitstyles
-npm install
+yarn
 ```
 
+It is recommended to use `yarn` instead of `npm install` to ensure version consistency across dependencies.
```

Comment:
Can you add a link to yarn here please? e.g. https://yarnpkg.com/en/docs/install